### ISSUE TITLE
Broken link : Readme Day 2- Accessibility & reproducibility

### ISSUE THREAD

**Issue by parisa-zahedi** [UtrechtUniversity/workshop-computational-reproducibility#111]
The link to Accessibility & reproducibility slides in Readme does not work
